### PR TITLE
Wait for status update after VM power operation

### PR DESCRIPTION
**Pull request by lhellebr**

### CODE REVIEW

File: tests/foreman/ui_airgun/test_computeresource.py [hunk 1]
```diff
@@ -267,6 +268,7 @@ def test_positive_resource_vm_power_management(
             session.computeresource.vm_poweroff(name, rhev_data['vm_name'])
         else:
             session.computeresource.vm_poweron(name, rhev_data['vm_name'])
+        time.sleep(30)  # wait for the VM's status to change to powered on/off
```
Comment (rplevka) [inline tests/foreman/ui_airgun/test_computeresource.py]: I think a retry loop with 1 sec delay will be a smarter idea. We'll be able to break just after the state change is detected. That way we only wait the necessary amount of time.

Comment (oshtaier) [reply tests/foreman/ui_airgun/test_computeresource.py]: I prefer to use `wait_for` library over retry loops

**Comment by oshtaier** [comment]
ping

**Comment by lhellebr** [comment]
I completely forgot about this one, sorry, will look into it soon

**Comment by lhellebr** [comment]
```
$ pytest tests/foreman/ui_airgun/test_computeresource.py::test_positive_resource_vm_power_management
===================================================== test session starts =====================================================
platform linux -- Python 3.6.8, pytest-3.6.1, py-1.7.0, pluggy-0.6.0
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/lhellebr/git/robottelo, inifile:
plugins: services-1.3.0, mock-1.10.0
collecting 2 items                                                                                                            2019-04-02 15:07:48 - conftest - DEBUG - Collected 2 test cases

collected 2 items                                                                                                             

tests/foreman/ui_airgun/test_computeresource.py ..                                                                      [100%]

================================================= 2 passed in 402.66 seconds ==================================================
```

**Pull request merged**